### PR TITLE
Fix 865

### PR DESCRIPTION
**Pull request by bpradipt**

### CODE REVIEW

**Review by snir911 — APPROVED** [review]
Cannot test but LGTM

**Comment by bpradipt** [comment]
Successfully tested docker build for Ubuntu

File: azure/image/Dockerfile [hunk 1]
```diff
@@ -70,5 +70,20 @@ RUN --mount=type=secret,id=AZURE_SUBSCRIPTION_ID \
     export AZURE_CLIENT_ID=$(cat /run/secrets/AZURE_CLIENT_ID) && \
     export AZURE_CLIENT_SECRET=$(cat /run/secrets/AZURE_CLIENT_SECRET) && \
     export AZURE_TENANT_ID=$(cat /run/secrets/AZURE_TENANT_ID) && \
```
Comment (mkulke) [inline azure/image/Dockerfile:72]: do we those still?

**Review by mkulke — APPROVED** [review]

File: azure/image/Makefile [hunk 2]
```diff
@@ -10,21 +10,18 @@ image: $(IMAGE_FILE)
...
-		-var az_image_name=${IMAGE_NAME} \
-		-var ssh_username=${SSH_USERNAME} \
-		-var publisher=${PUBLISHER} \
-		-var offer=${OFFER} \
-		-var sku=${SKU} \
-		-var plan_name=${PLAN_NAME} \
-		-var plan_product=${PLAN_PRODUCT} \
-		-var plan_publisher=${PLAN_PUBLISHER} \
-		./${PODVM_DISTRO}/
+	# export the required packer variables as PKR_VAR_<var_name>
```
[the remark this answers is not in the document]
Comment (bpradipt) [reply azure/image/Makefile:13]: Just added these comments for anyone directly looking at the Makefile and not the docs :-)

File: azure/image/Dockerfile (hunk 1 above)
Comment (bpradipt) [reply azure/image/Dockerfile:72]: Sorry I didn't get the question.

Comment (mkulke) [reply azure/image/Dockerfile:72]: 😅 me neither, probably something I didn't understand initiall. nevermind

**Pull request merged**